SimpleNtuplizer/plugins/SimpleNtuplizer.cc

```diff
@@ -438,6 +441,28 @@ SimpleNtuplizer::SimpleNtuplizer(const edm::ParameterSet& iConfig):
     photonTree_->Branch("genPdgId", &genPdgId_p);
     photonTree_->Branch("genStatus", &genStatus_p);
   }
+
+
+  ////PFtree
+  if(doPFTree){
+
+    pfTree_    = fs->make<TTree>("PfTree", "PF Cluster tree");
+    pfTree_->Branch("nClus_",   &nClus_);
+    
+    pfTree_->Branch("clusE_",   &clusE_);
+    pfTree_->Branch("clusPt_",   &clusPt_);
+    pfTree_->Branch("clusVx_",   &clusVx_);
+    pfTree_->Branch("clusVy_",   &clusVy_);
+    pfTree_->Branch("clusVz_",   &clusVz_);
+    pfTree_->Branch("clusEta_",   &clusEta_);
+    pfTree_->Branch("clusRho_",   &clusRho_);
+    pfTree_->Branch("clusPhi_",   &clusPhi_);
+    pfTree_->Branch("clusLayer_",   &clusLayer_);
+    pfTree_->Branch("clusNrecHits_",   &clusNrecHits_);
```

Comment:
I see that you didn't add any energy density variable. Are you sure that the rho that we have is available when making PFClusters? does it depend on the PF candidates to be calculated?